elements/pfe-modal/test/pfe-modal_test.html

```diff
@@ -51,7 +62,22 @@ <h2 slot="pfe-modal--header">Modal with a header</h2>
           assert.isTrue(modalWindow.hasAttribute('hidden'));
         
         });
-
+        
+        test('it should remove the hidden attribute from the host on upgrade', done => {
+          const pfeModal = document.querySelector('pfe-modal');
+          const modalWindow = pfeModal.shadowRoot.querySelector('.pfe-modal__window');
+          const trigger = pfeModal.querySelector('[slot=pfe-modal--trigger');
+          
+          // get the modal and fire the open function
```

Comment:
@LyndseyR This comment might need to be updated since we're not firing the o pen function but triggering a click event instead.